### PR TITLE
Upgrade fbjs, fbjs-scripts

### PR DESCRIPTION
**Pull request by zpao**
This just makes sure we're on the latest. This does come with a change that needs to be called out - that shallowCompare now treats NaN as the same value, and some other tiny equality related changes. This is because fbjs now uses Object.is to compare values. We talked about this a bit in #5773. The result here is that our "mutated style" warning will have fewer false positives and specifically will no longer warn for the NaN case (thus the updated test).

### CODE REVIEW

File: src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js [hunk 1]
```diff
@@ -198,16 +198,10 @@ describe('ReactDOMComponent', function() {
       ReactDOM.render(<span style={style}></span>, div);
```
Comment (gaearon) [inline src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js]: Might as well change the test title to skip `semi` :smile:

Comment (zpao) [inline src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js]: Good call. Want to accept too? :D

**Comment by jimfb** [comment]
:+1: 

How did the mutation warning go away?

**Comment by zpao** [comment]
shallowEqual now uses Object.is to compare values. We use shallowEqual in our style mutation detection.

**Pull request merged**